src/main/java/io/jenkins/plugins/oidc_provider/config/IdTokenConfiguration.java

```diff
@@ -155,6 +159,25 @@ public ListBoxModel doFillAlgorithmItems() {
         boolean result =  super.configure(req, json);
 
         // TODO update all credentials once the algorithm has changed
+        for (CredentialsStore store : CredentialsProvider.lookupStores(Jenkins.get())) {
```

Comment:
This will not cover folder credentials.

Again, I do not think you even need to bother with this, pending #3.

Reply:
Roger, I've removed it.